### PR TITLE
Implement self-test mode into attest-tool.

### PR DESCRIPTION
**Pull request by twitchy-jsonp**

### CODE REVIEW

File: attest/attest-tool/attest-tool.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ var (
 	nonceHex    = flag.String("nonce", "", "Hex string to use as nonce when quoting")
 	randomNonce = flag.Bool("random-nonce", false, "Generate a random nonce instead of using one provided")
 	useSHA256   = flag.Bool("sha256", false, "Use SHA256 for quote operatons")
+	ekIndex     = flag.Int("ek-index", 0, "Which EK to use for credential activation. The first EK is at index 0")
```
Comment (ericchiang) [inline attest/attest-tool/attest-tool.go]: nit: we don't guarantee ordering so drop this flag and just pick the first one?

Comment (twitchy-jsonp) [reply attest/attest-tool/attest-tool.go]: Done.

**Comment by twitchy-jsonp** [comment]
Github y u do dis

**Comment by twitchy-jsonp** [comment]
My guess is that force pushes automatically dismiss approval.

**Pull request merged**